sdk/core/azure-core/src/main/java/com/azure/core/credential/AccessToken.java

```diff
@@ -64,6 +82,17 @@ public OffsetDateTime getExpiresAt() {
         return expiresAt;
     }
 
+    /**
+     * Gets the time when the token should refresh, in UTC.
+     *
+     * Note: This value can be null.
```

Comment:
What does it mean to have a `null` value? Does it mean that token will never be refreshed or the next refresh time is unknown?

Reply:
Null means, there was no response from service here. 
So, it won't override the default refresh offset used in AccessTokenCache.
I can add this description to the javadocs.